frontend/src/components/common/FileUpload.js

```diff
@@ -1,13 +1,21 @@
 import { map } from "lodash";
 import React from "react";
+import PropTypes from "prop-types";
 import { FilePond, File, registerPlugin } from "react-filepond";
 import "filepond/dist/filepond.min.css";
 import FilePondPluginFileValidateSize from "filepond-plugin-file-validate-size";
-registerPlugin(FilePondPluginFileValidateSize);
+import FilePondPluginFileValidateType from "filepond-plugin-file-validate-type";
+registerPlugin(FilePondPluginFileValidateSize, FilePondPluginFileValidateType);
 
 import { ENVIRONMENT } from "@/constants/environment";
 import { createRequestHeader } from "@/utils/RequestHeaders";
 
+const propTypes = {
+  uploadUrl: PropTypes.string.isRequired,
+  maxFileSize: PropTypes.string,
```

Comment:
An optional prop requires a default value. Our linting rules will help spot things like this. If you haven't enabled the linter in your editor, I'd encourage you to do so, since I think everyone else working on the frontend has. This file, for example, has 9 linting errors and 1 warning.

If you have any questions about the linting, you can ask Justin and/or Ryan and/or read our contributions guidelines: https://github.com/bcgov/mds/tree/develop/frontend#code-contribution-standards

Thanks!

Reply:
Default props have been added, and some minor refactoring has been done to address linting issues.